acceptance-tests/src/test/java/me/atam/atam4jsampleapp/testsupport/Atam4jApplicationStarter.java

```diff
@@ -9,11 +9,12 @@
 public class Atam4jApplicationStarter {
 
     public static DropwizardTestSupport<ApplicationConfiguration> startApplicationWith(int initialDelayInMillis,
-                                                                                       Class testClass) {
+                                                                                       Class testClass, int periodInMillis) {
 
         ConfigOverride[] configOverrides = {
             ConfigOverride.config("testClasses", testClass.getName()),
-            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis))
+            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis)),
+            ConfigOverride.config("periodInMillis", String.valueOf(periodInMillis))
```

Comment:
I had the [open/closed principle](https://en.wikipedia.org/wiki/Open/closed_principle) in mind with the comment above. 

The principle is not easy to follow, but where possible I am inclined to keep classes and methods backwards compatible, i.e. it is ok to extend/add functionality, but _where possible_ changes to existing functionality (method signatures etc) should be avoided.